src/containers/DAO/pages/DAOList/TotalProvider.tsx

```diff
@@ -16,16 +18,20 @@ type TotalsProviderProps = {
 
 export const TotalsProvider = ({ children }: TotalsProviderProps) => {
 	const [totals, setTotals] = useState<Total[]>([]);
+	const [isLoading, setIsLoading] = useState<boolean>(true);
 
 	const add = (total: Total) => {
 		if (!totals.map((t) => t.zna).includes(total.zna)) {
 			setTotals([...totals, total]);
+			setIsLoading(false);
```

Comment:
nit: can get rid of this `setIsLoading`, as the second one will run regardless of whether or not the `if` condition is met